### PR TITLE
testing/lolcat: New aport

### PR DESCRIPTION
**Pull request by bratkartoffel**
This adds lolcat, a nice tool to brighten up your console.
Closes https://bugs.alpinelinux.org/issues/10504

### CODE REVIEW

File: testing/lolcat/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+pkgver=1.0
+pkgrel=0
+pkgdesc="High-performance implementation of lolcat" 
+url="https://github.com/jaseg/lolcat/" 
+arch="all" 
+license="WTFPL" 
+depends="" 
+makedepends="" 
+install="" 
+source="$pkgname-$pkgver.tar.gz::https://github.com/jaseg/${pkgname}/archive/v$pkgver.tar.gz
```
Comment (TBK) [inline testing/lolcat/APKBUILD]: Empty vars can be removed (depends, makedepends, install).

File: testing/lolcat/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+fix-url-and-version.patch" 
+
+build() {
+        make
+}
+
+check() {
+        ./lolcat --help 2>/dev/null
```
Comment (TBK) [inline testing/lolcat/APKBUILD]: We do not accept this kind of testing. So please remove check() and add `options="!check" # no test suite`

File: testing/lolcat/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+pkgrel=0
+pkgdesc="High-performance implementation of lolcat" 
+url="https://github.com/jaseg/lolcat/" 
+arch="all" 
+license="WTFPL" 
+depends="" 
+makedepends="" 
+install="" 
+source="$pkgname-$pkgver.tar.gz::https://github.com/jaseg/${pkgname}/archive/v$pkgver.tar.gz
+fix-url-and-version.patch"
```
Comment (TBK) [inline testing/lolcat/APKBUILD]:
```suggestion
    fix-url-and-version.patch
    "
```

**Comment by Ikke** [comment]
The commit message contains a typo: 'localt' -> 'lolcat'

**Comment by bratkartoffel** [comment]
Oh my, sorry for the poor quality of this PR. I've changed the suggestions, thanks for reviewing

**Pull request closed without merging**